statsmodels/base/distributed_estimation.py

```diff
@@ -0,0 +1,410 @@
+from statsmodels.base.elastic_net import fit_elasticnet
+from statsmodels.base.model import Results
+from statsmodels.tools.decorators import cache_readonly
+from statsmodels.regression.linear_model import OLS
+#from statsmodels.genmod.generalized_linear_model import GLM
+#from statsmodels.genmod.families import Binomial
+import statsmodels.base.wrapper as wrap
+import numpy as np
+
+"""
+distributed regularized estimation.
+
+Routines for fitting regression models using a distributed
+approach outlined in 
+
+Jason D. Lee, Qiang Liu, Yuekai Sun and Jonathan E. Taylor.
+"Communication-Efficient Sparse Regression: A One-Shot Approach." 
+arXiv:1503.04337. 2015.
+"""
+
+
```

Comment:
Currently everything here comes from the Taylor/Sun paper, I did use the Han Liu paper to confirm that I'm thinking about things correctly but there is nothing in the code that is distinct, do you think it is worth including the citation in that case?